tests/setup-files/setup.cjs

```diff
@@ -0,0 +1,3 @@
+const path = require('path')
+
+global.SETUP_CJS = path.basename(__filename) // using some cjs stuff here to make sure it works
```

Comment:
`globalThis` is the canonical name for the global object

Reply:
replaced with [globalThis](https://github.com/ExodusMovement/test/pull/12/commits/1f6169bd3f4f352397069be4b1613a0a3ada4827)